packages/Microsoft.SqlTools.ManagedBatchParser/Microsoft.SqlTools.ManagedBatchParser.nuspec

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>Microsoft.SqlTools.ManagedBatchParser</id>
+        <version>1.0.0</version>
+        <authors>Microsoft</authors>
+        <projectUrl>https://github.com/Microsoft/sqltoolsservice/</projectUrl>
+        <license type="expression">MIT</license>
+        <iconUrl>http://s.gravatar.com/avatar/6f39d8562df0a3509a8240fb281bc5bd?s=80</iconUrl>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>Managed Batch parser for Sql Tools Services.</description>
+        <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+        <tags>SQL Tools XPLAT Managed Batch Parser</tags>
+        <dependencies>
+            <group targetFramework="net472">
+                <dependency id="Microsoft.Data.SqlClient" version="5.0.1" />
+                <dependency id="Microsoft.SqlServer.SqlManagementObjects" version="170.8.0" />
+                <dependency id="Newtonsoft.Json" version="13.0.1" />
+                <dependency id="System.Configuration.ConfigurationManager" version="6.0.0" />
+            </group>
+            <group targetFramework="net6.0">
+                <dependency id="Microsoft.Data.SqlClient" version="5.0.1" />
+                <dependency id="Microsoft.SqlServer.SqlManagementObjects" version="170.8.0" />
+                <dependency id="Newtonsoft.Json" version="13.0.1" />
+                <dependency id="System.Configuration.ConfigurationManager" version="6.0.0" />
+            </group>
+        </dependencies>
+    </metadata>
+    <files>
+        <file src="..\..\artifacts\publish\Microsoft.SqlTools.ServiceLayer\default\net472\de\Microsoft.SqlTools.ManagedBatchParser.resources.dll" target="lib\net472\de" />
```

Comment:
>\net472\de\ [](http://example.com/codeflow?start=82&length=11)

FWIW, you should be able to use
   ...\net472\**\*.dll
and the like

Reply:
But we don't want all the net472 dlls

Reply:
If you used csproj you wouldn't have to do any of this, it'll pick up everything it needs by itself 😄

Reply:
As discussed in [comment](https://github.com/microsoft/sqltoolsservice/pull/1747#discussion_r1014412695) signed DLLs are not available in same output folder for packing with `no-build` using the same csproj file. This is old school design using build.cake targets, but shouldn't be a blocker for this PR.